File: source/testWebview/app/src/main/java/com/example/admin82/testwebview/MainActivity.java
package com.example.admin82.testwebview;

import android.net.Uri;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.KeyEvent;
import android.view.Menu;
import android.webkit.JavascriptInterface;
import android.webkit.WebView;
import android.webkit.WebViewClient;


public class MainActivity extends AppCompatActivity {


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        WebView myWebView = (WebView) findViewById(R.id.webView1);

        // javascriptを有効に
        myWebView.getSettings().setLoadWithOverviewMode(true);

        // スクロールバーの非表示
        myWebView.setScrollBarStyle(WebView.SCROLLBARS_INSIDE_OVERLAY);

        //リンクをタップしたときに標準ブラウザを起動させない
        myWebView.setWebViewClient(new WebViewClient());

        // 表示させたいページのリンク
        myWebView.loadUrl("http://help.g.pixiv-game.com/sp/");

        // http://help.g.pixiv-game.com/sp/
    }


    public boolean onKeyDown(int keyCode, KeyEvent event) {
        WebView myWebView = (WebView) findViewById(R.id.webView1);
        // 端末のBACKキーで一つ前のページヘ戻る
        if (keyCode == KeyEvent.KEYCODE_BACK && myWebView.canGoBack()) {
            myWebView.goBack();
            return true;
        }
        return super.onKeyDown(keyCode, event);
    }
}

File: source/URLlinkApp/app/src/main/java/com/example/admin82/urllinkapp/MainActivity.kt
package com.example.admin82.urllinkapp

import android.net.Uri
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.content.Intent
import android.os.Process.*
import android.view.View
import android.widget.TextView
import android.widget.Button
import android.view.View.OnClickListener
import android.content.UriPermission
import android.webkit.WebView


class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        // URIを取得
        val uri = Uri.parse("http://help.g.pixiv-game.com/sp/")
        // Intent処理を呼び出す
        val intent = Intent(Intent.ACTION_VIEW, uri)
        // 指定ページに飛ぶ
        startActivity(intent)

        // 起動アプリを閉じる
        finish()
    }


}
